### PR TITLE
style: global formatting using the nph tool

### PR DESCRIPTION
**Pull request by Ivansete-status**
## Description
Simple PR from where we start using the nph formatter in VSCode.

EDITED: The change is pretty generic but it was very simple to do thanks to the following smart script: https://github.com/arnetheduck/nph/blob/537f1ce6e30df1b915cac45538e7d0de7dd0b853/format-git-repo.sh

### CODE REVIEW

**Comment by arnetheduck** [comment]
hold off a few days, 0.4 is coming

**Comment by arnetheduck** [comment]
also, see https://github.com/arnetheduck/nph/blob/master/format-git-repo.sh and https://arnetheduck.github.io/nph/faq.html#how-do-i-introduce-nph-in-an-existing-codebase

**Comment by arnetheduck** [comment]
...and finally - make sure to review the formatting manually - there are still some oddities probably - if you find anything, it's good to report them to the nph repo

File: waku/waku_discv5.nim
Comment (SionoiS) [inline waku/waku_discv5.nim:71]: Inline comments are going to be scrambled. Not sure what to do about that.

File: waku/waku_metadata/protocol.nim
Comment (SionoiS) [inline waku/waku_metadata/protocol.nim:16]: Not sure I like this style. Seams harder to read than vertical

Comment (Ivansete-status) [reply waku/waku_metadata/protocol.nim:16]: Yes, I agree that is easier to read in vertical style. Also better for comparing. We can submit a request to the `nph` repo.

File: waku/waku_discv5.nim
Comment (Ivansete-status) [reply waku/waku_discv5.nim:71]: Good catch!
That sounds more like a tiny bug. I think we cannot merge until this is sorted out. I will submit it

File: apps/chat2bridge/chat2bridge.nim
Comment (AlejandroCabeza) [inline apps/chat2bridge/chat2bridge.nim:241]: Another comment that could be considered scrambled.

File: waku/waku_metadata/protocol.nim
Comment (AlejandroCabeza) [reply waku/waku_metadata/protocol.nim:16]: I kinda think the same. Vertical and sorted by name/directory would be great and/or grouped.

File: apps/chat2bridge/chat2bridge.nim
Comment (AlejandroCabeza) [inline apps/chat2bridge/chat2bridge.nim:241]: Another comment that could be considered scrambled.

File: library/waku_thread/inter_thread_communication/requests/node_lifecycle_request.nim
Comment (Ivansete-status) [inline library/waku_thread/inter_thread_communication/requests/node_lifecycle_request.nim:2]: It's been commented on elsewhere but this should be vertical IMO

File: tests/node/test_wakunode_store.nim
Comment (Ivansete-status) [inline tests/node/test_wakunode_store.nim:312]: This inline comment should be kept inline too :)

File: tests/test_waku_discv5.nim
Comment (Ivansete-status) [inline tests/test_waku_discv5.nim:398]: Maybe already commented. Better in vertical mode

**Review by Ivansete-status — COMMENTED** [review]
I just add some points that we need to consider as possible enhancements.

File: apps/chat2bridge/chat2bridge.nim
Comment (arnetheduck) [reply apps/chat2bridge/chat2bridge.nim:241]: this formatting is "correct": it would not fit on the same line, and in order to show that it "belongs" to `pop` (and not whatever comes after), it's indented.

File: library/waku_thread/inter_thread_communication/requests/node_lifecycle_request.nim
Comment (arnetheduck) [reply library/waku_thread/inter_thread_communication/requests/node_lifecycle_request.nim:2]: you can make it vertical in several ways:
* use a single import statement that gets "too long" and "too complex"
* document the imports:

```
import 
  chronos # bla bla
  abc # bla bla
```

when imports are "simple", we use a simple formatting as well (many in the community, confirmed by looking at how imports are formatted in the wild), prefer not to have them spread out too much, when it's just a few

File: tests/node/test_wakunode_store.nim
Comment (arnetheduck) [reply tests/node/test_wakunode_store.nim:312]: when a comment doesn't fit the 88-char limit, we generally move it to a new line and indent it (to highlight that it "belongs" to the previous, less indented item)

Comment (arnetheduck) [reply tests/node/test_wakunode_store.nim:312]: see the "assert" just a few lines above for a more obvious example  of this same principle

File: apps/chat2/config_chat2.nim
Comment (arnetheduck) [reply apps/chat2/config_chat2.nim:24]: the formatting rule is "if it fits on a single line, use a single line" - without this rule, trivial pragmas (and lists of numbers etc) would be spread over multiple lines which is very distracting.

**Comment by alrevuelta** [comment]
@Ivansete-status Is there anything blocking this PR?

**Comment by Ivansete-status** [comment]
> @Ivansete-status Is there anything blocking this PR?

Thanks for the comment!

I reviewed it completely and I didn't see major issues from my side (`v0.4.1`.). The only future enhancement would be that I'd like it to adopt a more vertical approach, specially in imports and object ctors, but this is very complex to achieve in a general way (cc @arnetheduck.)

@alrevuelta - Feel free to review the formatting results and approve or comment if you are happy with them.

@SionoiS , @AlejandroCabeza - are you happy with the resulting format ?

If we all agree that want to go ahead with the `nph`, I'll force-push the PR with latest code base state.
If we are not 100% confident, then is a good opportunity to comment the points with Jacek, and we can postpone this refactoring to the future.

**Review by jm-clius — APPROVED** [review]
LGTM! :)

**Review by SionoiS — APPROVED** [review]
Looked at some part I know well and it is beautiful!

Thank you!

**Review by gabrielmer — APPROVED** [review]
Soooo good! Thanks so much!

**Pull request merged**
